rocky/assets/css/themes/soft/manon/critical-informational-labels.scss

```diff
@@ -1,19 +1,23 @@
 /* Critical - informational labels - Variables */
 
-.recommendation {
-  padding: var(--critical-informational-labels-padding);
-  min-width: var(--critical-informational-labels-min-width);
-  text-align: var(--critical-informational-labels-text-align);
-  display: inline-block;
-  background-color: #c3ddf6;
-}
+// .recommendation {
+//   padding: var(--critical-informational-labels-padding);
+//   min-width: var(--critical-informational-labels-min-width);
+//   text-align: var(--critical-informational-labels-text-align);
+//   display: inline-block;
+//   background-color: #c3ddf6;
+// }
 
-table .critical,
-table .high,
-table .medium,
-table .low,
-table .informational,
-table .recommendation {
-  width: 100%;
-  box-sizing: border-box;
-}
+// table .critical,
+// table .high,
+// table .medium,
+// table .low,
+// table .informational,
+// table .recommendation {
+//   width: 100%;
+//   box-sizing: border-box;
+// }
+
+// table .medium {
+//   border: 1px dashed black;
+// }
```

Comment:
The file doesn't have any content anymore that is not commented out and isn't included anymore as far as I can see, so should't it just be deleted?

Reply:
Ah yes. Good catch I've removed it. Thanks